azurerm/resource_arm_netapp_account.go

```diff
@@ -0,0 +1,273 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/netapp/mgmt/2019-06-01/netapp"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+	aznetapp "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/netapp"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmNetAppAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmNetAppAccountCreateUpdate,
+		Read:   resourceArmNetAppAccountRead,
+		Update: resourceArmNetAppAccountCreateUpdate,
+		Delete: resourceArmNetAppAccountDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: aznetapp.ValidateNetAppAccountName,
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
+
+			"location": azure.SchemaLocation(),
+
+			"active_directory": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"dns_servers": {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem: &schema.Schema{
+								Type:         schema.TypeString,
+								ValidateFunc: aznetapp.ValidateActiveDirectoryDNSName,
```

Comment:
Does this only support ipv4 values? i would have assumed a FQDN would work as well. `dns.hashicorp.com`?

If it is only IPv4 addresses that are allowed could we use the existing IPv4 validator?

Reply:
After checked, it doesn't support IPv6 and FQDN. So I've updated code to use the existing IPv4 validator. Thanks.
![image](https://user-images.githubusercontent.com/19754191/67255284-de745500-f4b3-11e9-9244-14d5c72a3f8f.png)
![image](https://user-images.githubusercontent.com/19754191/67255287-e2a07280-f4b3-11e9-9c41-46a5dadca52c.png)

Reply:
After confirmed with the service owner, they also confirmed only IPv4 is permitted.